### PR TITLE
Rbn 40 create and fit footer

### PR DESCRIPTION
**Pull request by maxencije**
### What did you implement?

Footer component is implemented, fully responsive.

### More info

- Jira: [RBN-40](https://productdock.atlassian.net/browse/RBN-40)

- Figma: [Design](https://www.figma.com/design/x4SgW80BE24s9gAFHM3gXS/RBC---Novi-Sad%2C-2024?node-id=2597-13715&t=zQZFMBZ56c1sxpri-4)

### CODE REVIEW

File: src/components/Footer/BackToTop.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import Arrow from '../../assets/arrow.png'
+import styles from './Footer.module.css'
+
+const BackToTop = () => {
+  const scrollToTop = () => {
+    window.scroll(0, 0)
```
Comment (ssukovic) [inline src/components/Footer/BackToTop.tsx]: Add `smooth` behaviour to scroll, so it's not so harsh. See how [here](https://developer.mozilla.org/en-US/docs/Web/API/Window/scroll).

File: src/assets/footerLogo.svg
Comment (ssukovic) [inline src/assets/footerLogo.svg:1]: Remove one of the stamp svgs

File: src/assets/arrow.png
Comment (ssukovic) [inline src/assets/arrow.png:1]: Export this icon as svg from figma for better quality. SVG images are vector based which allows resizing without losing quality. This is how it looks like now: 
Screenshot 2024-10-03 at 12 59 52

**Review by ssukovic — CHANGES_REQUESTED** [review]

**Review by ssukovic — DISMISSED** [review]
🚀

**Review by Aleksandra-Romanic — DISMISSED** [review]

**Review by ssukovic — APPROVED** [review]

**Review by isidora-b — APPROVED** [review]

**Pull request merged**